numpy/lib/shape_base.py

```diff
@@ -255,6 +255,7 @@ def expand_dims(a, axis):
 
     See Also
     --------
+    squeeze : The inverse operation, removing singleton dimensions
```

Comment:
Might also mention `reshape` in both contexts as it allows adding and removing axis.